tests/fuzz/fuzz.go

```diff
@@ -0,0 +1,38 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build gofuzz
+
+package fuzz
+
+import (
+	"github.com/istio/istio/pilot/pkg/config/kube/crd"
+	"github.com/istio/istio/pkg/config/schema"
+)
+
+func FuzzParseInputs(data []byte) int {
+	_, _, err := crd.ParseInputs(string(data))
+	if err != nil {
+		return 0
+	}
+	return 1
+}
+
+func FuzzParseAndBuildSchema(data []byte) int {
+	_, err := schema.ParseAndBuild(string(data))
```

Comment:
What is the "cost" of each of these? For example, a unit test like this would run in microseconds, so even if its not super useful its not a high cost. Is this going to be eating up substantial CPU/time/$$?

I am asking in general to get a feel since I am not super familiar with fuzzing, but also this function is only used at build time so its probably low priority.

Reply:
The cost of an execution varies from target to target. Some applications require lots of initialization and executes less than 10 times per second. Other fuzzers execute 100.000 times per second. In the case of the ParseAndBuild-fuzzer, it has about 8000 executions per second on my local machine.